packages/editor-ui/src/components/BecomeTemplateCreatorCta/becomeTemplateCreatorStore.ts

```diff
@@ -0,0 +1,94 @@
+import { DateTime } from 'luxon';
+import { defineStore } from 'pinia';
+import { computed, ref } from 'vue';
+import { STORES } from '@/constants';
+import { useCloudPlanStore } from '@/stores/cloudPlan.store';
+import { useStorage } from '@/composables/useStorage';
+import { useRootStore } from '@/stores/n8nRoot.store';
+import { getUserCtas } from '@/api/ctas';
+
+const LOCAL_STORAGE_KEY = 'BECOME_TEMPLATE_CREATOR_CTA_DISMISSED_AT';
+const RESHOW_DISMISSED_AFTER_DAYS = 30;
```

Comment:
From what I recall all other banners are dismissed permanently, so this insistence might get annoying...